src/App.svelte

```diff
@@ -12,13 +22,83 @@
 			""		// Empty space
 		]
 	}
+	
+	let tableBody = {"bodys": [
+		{
+			"unique_id": "10", 
+  			"title": "How to Add",
+  			"username": "lronhubbard",
+  			"timestamp": "20211213T135800", 
+  			"tag": "addition",
+			"content": "How does add work? I don't understand 3+3=6?"
+		},
+		{
+			"unique_id": "11", 
+			"title": "How to Subtract", 
+			"username": "lronhubbard", "timestamp": 
+			"20211213T135801", "tag": "subtraction", 
+			"content": "How does subtract work? I don't understand 3-3=0?"
+		}
+		]
+	}
+
+	let postContents;
+	let comment_list;
+	function getpost(event) {
+		selected.component = "Post";
+
+		postContents = {
+			"unique_id": event.detail.unique_id,
+			"title": event.detail.title,
+			"username": event.detail.username,
+			"timestamp": event.detail.timestamp,
+			"tag": event.detail.tag,
+			"content": event.detail.content
+		};
+
+		comment_list = {"item": [
+			{
+				"unique_comment_id": "1",
+				"thread_id": "10",
+				"username": "uwuwuwu",
+				"timestamp": "20211213T143800",
+				"content": "why are you here :thonk:"
+			},
+			{
+				"unique_comment_id": "2",
+				"thread_id": "10",
+				"username": "lronhubbard",
+				"timestamp": "20211215T153800",
+				"content": "baka baka"
+			}
+		]
+		};
+	}
+
+	function goback(){
+		selected.component = "Table";
+	}
 </script>
 
+
 <main>
 	<h1>Hello {name}!</h1>
 	<p>Visit the <a href="https://svelte.dev/tutorial">Svelte tutorial</a> to learn how to build Svelte apps.</p>
+	
```

Comment:
We probably want to have high-level components for "landing page view", "thread list view", and "thread detail + comments view". Then we can just juggle those three components when state changes rather than manually going and creating/destroying a bunch of components. *Let me know if I need to explain more!*